pkg/controller/secret/secret_controller.go

```diff
@@ -253,7 +260,7 @@ func addPDSecretToAlertManagerConfig(r *ReconcileSecret, request *reconcile.Requ
 			"severity",
 		},
 		MatchRE: map[string]string{
-			"namespace": "^openshift$|openshift-.*|default$|kube$|kube-.*|logging$",
+			"namespace": "^kube-.*,^openshift-.*,^logging$,^default$,^openshift$,^ops-health-monitoring$,^ops-project-operation-check$,^management-infra$",
```

Comment:
The following namespaces will not be created by SRE in OSD v4:

- ops-health-monitoring
- ops-project-operation-check
- management-infra

But, we blacklist these with dedicated-admin operator:
https://github.com/openshift/dedicated-admin-operator/blob/master/pkg/dedicatedadmin/dedicatedadmin.go#L53

So there is no risk unless we change the regex for dedicated-admin operator.